Formula/openimageio.rb

```diff
@@ -0,0 +1,158 @@
+  end
+
+  option "with-test", "Dowload 95MB of test images and verify Oiio (~2 min)"
+  option "with-jpeg-turbo", "Build with libjpeg-turbo support, instead of libjpeg"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "boost"
+  depends_on "openssl"
+  depends_on "cfitsio"
+  depends_on "field3d"
+  depends_on "freetype"
+  depends_on "giflib"
+  depends_on "hdf5"
+  depends_on "ilmbase"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "libtiff"
+  depends_on "opencolorio"
+  depends_on "openexr"
+  depends_on "openjpeg"
+  depends_on "webp"
+  depends_on "ffmpeg" => :recommended
+  depends_on "libraw" => :recommended
+  depends_on "ptex" => :recommended
+  depends_on "opencv@2" => :recommended
+  depends_on "python3" => :optional
+  depends_on "jpeg-turbo" => :optional
+
+  depends_on "boost-python" => (build.with?("python3") ? ["with-python3", "without-python"] : [])
+
+  resource "j2kp4files" do
+    url "http://pkgs.fedoraproject.org/repo/pkgs/openjpeg/j2kp4files_v1_5.zip/27780ed3254e6eb763ebd718a8ccc340/j2kp4files_v1_5.zip"
+    sha256 "21cf3156ed2a2a39765c0d57f36c71d1291e9c30054775a2f0a8fdd2964f1799"
+  end
+
+  # DNS cannot resolve ftp.remotesensing.org to 140.211.15.132
+  resource "tiffpic" do
+    url "ftp://140.211.15.132/pub/libtiff/pics-3.8.0.tar.gz"
+    mirror "http://dl.maptools.org/dl/libtiff/pics-3.8.0.tar.gz"
+    mirror "http://ftp.ntua.gr/pub/graphics/tiff/pics-3.8.0.tar.gz"
+    mirror "ftp://ftp.u-aizu.ac.jp/pub/graphics/image/ImageMagick/simplesystems.org/libtiff/pics-3.8.0.tar.gz"
+    sha256 "e0e34732b61e1ce49eff2c7a079994c856d2a5f772f5228c84678272bc6829a9"
+  end
+
+  resource "bmpsuite" do
+    url "http://entropymine.com/jason/bmpsuite/bmpsuite.zip"
+    sha256 "7c7643003476da4e2b29ebbb0ed1ca28cf7eb21a04b4f474567c1bea5caba089"
+    version "1.0.0"
+  end
+
+  resource "tgautils" do
+    url "https://github.com/dunn/mirrors/raw/master/openimageio/TGAUTILS.ZIP"
+    sha256 "1c05c376800d75332e544b665354b9e234f97352266b4dea40d5424d8bcb3299"
+    version "1.0.0"
+  end
+
+  resource "openexrimages" do
+    url "https://download.savannah.nongnu.org/releases/openexr/openexr-images-1.7.0.tar.gz"
+    sha256 "99e3fabef4672f62f4a10a470eea4a161026d488cabc418fff619638deacf807"
+  end
+
+  resource "oiioimages" do
+    url "https://github.com/OpenImageIO/oiio-images.git",
+        :revision => "9a70c65c7a29a50114a8208d61c87ba4fedd018e"
+  end
+
+  def install
+    # Oiio is designed to have its testsuite images extracted one directory
+    # above the source.  That's not a safe place for HB.  Do the opposite,
+    # and move the entire source down into a subdirectory if --with-test.
+    if build.with? "test"
+      (buildpath+"localpub").install Dir["*"]
+      chdir "localpub"
+    end
+
+    ENV.append "MY_CMAKE_FLAGS", "-Wno-dev" # stops a warning.
+    ENV.append "MY_CMAKE_FLAGS", "-DUSE_OPENCV=OFF" if build.without? "opencv"
```

Comment:
Remove, there is no "opencv" option anyway

Reply:
It's `opencv@2`

Reply:
Yeah, so it should have been `opencv@2` here.
But it is going to be mandatory, so the line can be removed anyway

Reply:
That's the question. I suggest to make opencv optional since it installs a hole bunch of other dependencies (numpy, python) and is used barely in openimageio (Ipl image format and image capture). Therefore I would prefer to have it removed from the bottle build.

Reply:
Then either optional, or removing it entirely if it's not used much.